int32 trynumber
---
bool detectedFace
